ErrorBlocks.md

```diff
@@ -0,0 +1,156 @@
+
+<details><p><summary>Notes</summary>
+- Most often in real models unnecessary details must be determined intuitively and visually while overlaps, edge contacts, and small gaps cannot be found visually.
+<br/></br>
+- For all of these edits one must work closely with the creator of the model/experiment to determine which complications are important to the end result and which are able to be changed.
+</p></details>
+
+### In Cubit/Trelis
+_Cubit and Trelis are basically the same CAD software with Trelis being the commercial version. They are most useful for the meshing of geometries and assigning materials to volumes._
+
+| Steps  | Graphic User Interface  | In Command Line  |
+|---|---|---|
+| __Import the .sat file into Trelis/Cubit__  | - Can be imported using the GUI with __File -> Import__. <br/></br> - Select the file through file explorer and open. <br/></br> - One important qualifier to include is `[attributes_on]`. | - In the command line the simple form of the command is `import acis [filepath]` <br/></br> - This command can be appended with further commands that can be found by typing `help import acis`. <br/></br> - These qualifiers correspond with the options given when importing files through the GUI. <br/></br> - One important qualifier to include is `[attributes_on]`.  |
+| __How to save progress for the file as ACIS__  | - Click on file, export, and select the file destination and name. <br/></br> - Make sure that it is being exported as ACIS (.sat) and press ok to save.  | - In command line type `export acis [filepath] overwrite`.  |
+| __Make sure units/distances are correct and if not, fix them__  | - A distance within the model of "X units" should have a metric distance of "X centimeters". <br/></br> - To check this click on a surface/line with a known metric value and then navigate to the properties page and check the surface area or curve length. <br/></br> - On the Command Panel click on __Geometry (leftmost option) -> Volume -> Transform -> Scale (from dropdown menu) -> Enter Scale Factor -> Apply__ | - A distance within the model of "X units" should have a metric distance of "X centimeters". <br/></br> - To check this in the command line the general form is `list [entity name and number] geom`. For example to check the geometry of volume 1 enter `list Volume 1 geom`. <br/></br> - In command line enter `vol all scale [Scale Factor]`. For example parts from solidworks would require `vol all scale 0.1` to convert the units to centimeters. |

```

Comment:
"`vol all scale 0.1` to convert the units to centimeters" doesn't necessarily convert to cm does it? Rather this literally scales it to one tenth the original size. The way this reads right now might lead someone to believe that regardless of their starting scaling, if they use 0.1, it will automatically go to cm scaling, which is not necessarily correct because it all depends on the size of the model when it is imported.